homeassistant/components/frontend/__init__.py

```diff
@@ -81,7 +94,26 @@
         vol.Optional(CONF_EXTRA_HTML_URL_ES5):
             vol.All(cv.ensure_list, [cv.string]),
         vol.Optional(CONF_JS_VERSION, default=JS_DEFAULT_OPTION):
-            vol.In(JS_OPTIONS)
+            vol.In(JS_OPTIONS),
+        vol.Optional(CONF_MANIFEST_JSON, default={
+            CONF_MANIFEST_JSON_DISPLAY: CONF_MANIFEST_JSON_DISPLAY_DEFAULT,
+            CONF_MANIFEST_JSON_SHORT_NAME:
+                CONF_MANIFEST_JSON_SHORT_NAME_DEFAULT,
+            CONF_MANIFEST_JSON_START_URL: CONF_MANIFEST_JSON_START_URL_DEFAULT
+        }): vol.Schema({
+            cv.string: cv.string,
+            vol.Optional(
+                CONF_MANIFEST_JSON_DISPLAY,
+                default=CONF_MANIFEST_JSON_DISPLAY_DEFAULT
+            ):
+            vol.In(MANIFEST_JSON_DISPLAY_OPTIONS),
+            vol.Optional(
+                CONF_MANIFEST_JSON_SHORT_NAME,
+                default=CONF_MANIFEST_JSON_SHORT_NAME_DEFAULT): cv.string,
+            vol.Optional(
```

Comment:
Why would we want to make start url configurable ?

Reply:
I have read some posts in the community forum of users wanting to customize the landing page to use custom components. Actually I have reviewed the full list of manifest.json possible options and these 3 are the only ones I think should be user configurable. I do not think letting users configure this could become a problem but you have the bigger picture and experience so it is your call.

Reply:
Since Lovelace is fully configurable, I wonder if that is still the case.

Reply:
I agree that besides the display setting the other ones do not help very much.